pkg/operator/ceph/csi/config.go

```diff
@@ -136,22 +130,38 @@ func generateProfileSubVolumeGroupSpec(clusterInfo *cephclient.ClusterInfo, ceph
 // CreateDefaultClientProfile creates a default client profile for csi-operator to connect driver
 func CreateDefaultClientProfile(c client.Client, clusterInfo *cephclient.ClusterInfo, namespaced types.NamespacedName) error {
 	logger.Info("Creating ceph-csi clientProfile default CR")
+	defer func() {
+		if r := recover(); r != nil {
+			logger.Errorf("HIT A PANIC WHEN CREATING DEFAULT CLIENT PROFILE: %+v", r)
```

Comment:
Of course we don't expect to experience any more panics, but perhaps we should keep this recover for now just in case
```suggestion
			logger.Errorf("Panic when creating the default client profile: %+v", r)
```